internal/resources/metal/project/resource_schema.go

```diff
@@ -93,6 +93,9 @@ var bgpConfigSchema = map[string]schema.Attribute{
 	"asn": schema.Int64Attribute{
 		Description: "Autonomous System Number for local BGP deployment",
 		Required:    true,
+		PlanModifiers: []planmodifier.Int64{
+			int64planmodifier.RequiresReplace(),
```

Comment:
This would result in replacing the entire project, right?  I'm not sure if we should take that risk or if it's better to reject the plan via validation.

Reply:
Relates to:
* https://github.com/equinix/terraform-provider-equinix/issues/160
* https://github.com/packethost/terraform-provider-packet/issues/299

Reply:
agree that reject the change will be better in this case